File: ios/ZhengQi/Models/Quote.swift
import Foundation
import SwiftData

@Model
final class Quote {
    var id: UUID
    var content: String
    var author: String
    var category: String
    var isFavorite: Bool

    init(id: UUID = UUID(), content: String, author: String, category: String = "修心", isFavorite: Bool = false) {
        self.id = id
        self.content = content
        self.author = author
        self.category = category
        self.isFavorite = isFavorite
    }
}

File: android/core/src/main/java/com/zhiguan/app/data/model/Quote.kt
package com.zhiguan.app.data.model

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "quotes")
data class Quote(
    @PrimaryKey(autoGenerate = true) val id: Long = 0,
    val content: String,
    val author: String,
    val category: String = ""
)